extensions/amp-story/1.0/amp-story-desktop-panels.css

```diff
@@ -132,3 +140,10 @@ amp-story[standalone].i-amphtml-story-desktop-panels {
   min-width: 320px !important;
   min-height: 533px !important;
 }
+
+.i-amphtml-story-desktop-panels .i-amphtml-story-button-container {
+  height: calc(75vh * .9) !important;
```

Comment:
should these use the `--i-amphtml-story-inactive-page-scale-factor` too?

Reply:
Yes! Thank you for catching this.